frameworks/remix-run/application/src/routes/$langstore.search.tsx

```diff
@@ -26,6 +27,6 @@ export const loader: LoaderFunction = async ({ request }) => {
 };
 
 export default () => {
-    const { data } = useLoaderData();
+    const { data } = useLoaderData() as { data: ProductSlim[] };
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`